src/Pages/TeamPage/TeamPage.module.scss

```diff
@@ -0,0 +1,215 @@
+$fontcolor: #ffffff;
+
+.teamsCont {
+  width: 100%;
+  overflow-x: hidden;
```

Comment:
this overflow hidden is hampering the functioning of  react-scroll. so either remove this or don't use react-scroll.